config/php-cs-fixer.php

```diff
@@ -20,59 +21,102 @@
             '@PHPUnit60Migration:risky' => true,
             '@PHPUnit75Migration:risky' => true,
             '@PSR12' => true,
+
+            // alias
+            'no_alias_functions' => true,
+
+            // array notation
             'array_syntax' => ['syntax' => 'short'],
-            'blank_line_after_opening_tag' => true,
-            'cast_spaces' => ['space' => 'none'],
-            'class_attributes_separation' => true,
-            'combine_consecutive_issets' => true,
-            'combine_consecutive_unsets' => true,
-            'compact_nullable_typehint' => true,
-            'concat_space' => ['spacing' => 'one'],
-            'declare_equal_normalize' => true,
-            'declare_strict_types' => true,
-            'dir_constant' => true,
-            'echo_tag_syntax' => true,
-            'elseif' => true,
+            'no_trailing_comma_in_singleline_array' => true,
+            'no_whitespace_before_comma_in_array' => true,
+            'whitespace_after_comma_in_array' => true,
+
+            // basic
             'encoding' => true,
-            'escape_implicit_backslashes' => ['single_quoted' => true],
-            'function_typehint_space' => true,
-            'is_null' => true,
-            'linebreak_after_opening_tag' => true,
-            'lowercase_cast' => true,
+            'psr_autoloading' => true,
+
+            // casing
             'magic_constant_casing' => true,
-            'modernize_types_casting' => true,
-            'multiline_whitespace_before_semicolons' => true,
             'native_function_casing' => true,
-            'native_function_invocation' => ['include' => ['@all']],
-            'new_with_braces' => true,
-            'no_alias_functions' => true,
+
+            // cast notation
+            'cast_spaces' => ['space' => 'none'],
+            'lowercase_cast' => true,
+            'modernize_types_casting' => true,
+            'no_short_bool_cast' => true,
+            'short_scalar_cast' => true,
+
+            // class notation
+            'class_attributes_separation' => true,
             'no_blank_lines_after_class_opening' => true,
-            'no_blank_lines_after_phpdoc' => true,
-            'no_empty_comment' => true,
-            'no_empty_phpdoc' => true,
-            'no_empty_statement' => true,
-            'no_extra_blank_lines' => true,
-            'no_leading_import_slash' => true,
-            'no_leading_namespace_whitespace' => true,
             'no_php4_constructor' => true,
-            'no_short_bool_cast' => true,
-            'no_singleline_whitespace_before_semicolons' => true,
-            'no_spaces_after_function_name' => true,
-            'no_spaces_inside_parenthesis' => true,
+
+            // class usage
```

Comment:
Missing comment explaining comment.

Reply:
Thanks, missed those two. Fixed and repushed.